src/modal/modal.component.ts

```diff
@@ -112,7 +117,7 @@ export class Modal implements AfterViewInit, OnInit, OnDestroy {
 	/**
 	 * Size of the modal to display.
 	 */
-	@Input() size: "sm" | "md" | "lg" | "xl" | "xxl" = "md";
+	@Input() size: "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
```

Comment:
We don't support an `xxl` size right? We should be able to drop that

Reply:
Or at the very least mention that we don't actually support `xxl`

Reply:
It would be good to reach some level of parity with [Carbon](https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/components/modal/_modal.scss) and the [React implementation](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Modal/Modal.js#L138) here. In other words, this might be an opportunity to remove non-functional code without introducing breaking changes while aligning with Carbon (the source of truth) and a similar component implementation.

E.g. the modal has a default size from `bx--modal-container` (what this calls "md") and three size variants ("xs", "sm", and "lg") as the _only_ options for `size`. If a developer specifies a non-Carbon size it could just default to the default size like always.

I'm also not sure it's safe to assume that "md" is the default for all components (e.g. I would expect a Carbon button size variant that sits between the default and "sm" would be "md"), but that's a topic for a different issue!

Reply:
Great points @patricklowry ... since this code was non-functional until this point, It shouldn't be a breaking change to simplify the type 👍